WebDriverAgentTests/IntegrationTests/FBPasteboardTests.m

```diff
@@ -41,7 +41,8 @@ - (void)testSetPasteboard
   XCTAssertTrue([textField fb_clearTextWithError:&error]);
   [textField pressForDuration:2.0];
   XCUIElement *pasteItem = [[self.testedApplication descendantsMatchingType:XCUIElementTypeAny]
-                            matchingIdentifier:@"Paste"].fb_firstMatch;
+                           // fb_firstMatch does not work on iOS 13
+                            matchingIdentifier:@"Paste"].firstMatch;
```

Comment:
`fb_firstMatch` returns `nil`, but `firstMatch` returns expected object.
ah..

Reply:
https://github.com/appium/WebDriverAgent/blob/master/WebDriverAgentLib/Utilities/FBXCodeCompatibility.m#L109
When I wait a bit here, this works with `fb_firstMatch` since the `exits` returns true. But without break point, the `exists` return false....

Reply:
How about waiting for element existence? : https://developer.apple.com/documentation/xctest/xcuielement/2879412-waitforexistencewithtimeout?language=objc

Reply:
Worked 👍 
1 sec was enough on my env. The wait time when the first match failed is also almost the same as `exists` for now. Pure `firstMatch` is probably 1 ~ 2 sec. (comparing the fails time)